commit 2d76c145c35955c827e401b86d1f55fd9fa8d893
Author: wm4 <wm4@nowhere>
Date:   Fri Jun 3 20:03:49 2016 +0200

vo_opengl: fix giant memory leaks with icc profiles

Well this was dumb.
---

diff --git a/video/out/opengl/video.c b/video/out/opengl/video.c
index 1f053d4..c93bb67 100644
--- a/video/out/opengl/video.c
+++ b/video/out/opengl/video.c
@@ -649,6 +649,8 @@ static bool gl_video_get_lut3d(struct gl_video *p, enum mp_csp_prim prim,

     debug_check_gl(p, "after 3d lut creation");

+    talloc_free(lut3d);
+
     return true;
 }

